README.md

```diff
@@ -32,7 +32,15 @@ defined, so you don't have to import them (see examples below).
 See [octokit/rest.js](https://octokit.github.io/rest.js/) for the API client
 documentation.
 
-## Breaking changes in V5
+## Breaking Changes
+
+### Breaking changes in V6
+
+Version 6 of this action updated the runtime to Node 16 - https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-using-nodejs-v16
+
+All scripts are now run with Node 16 instead of Node 12 and are affected by any breaking changes between Node 12 and 16.
```

Comment:
Maybe link to the release notes?  https://nodejs.org/en/download/releases/

There's not really any explicit list of breaking changes, so it might not be useful though

Reply:
The release notes are quite large (ex https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V16.md), so I'm reluctant to link to them as I don't think they'd be useful in documenting only the breaking changes.